microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -477,24 +491,47 @@ When loading content via a file URL or a virtual host name mapping, the resoluti
 This can take some time. Make sure to limit calls to `AddWebResourceRequestedFilter` to only the web resources that must raise the `WebResourceRequested` event.
 
 
+<!-- ---------- -->
+###### Source maps with the `WebResourceRequested` event
+
+Source maps are needed to debug the source code of compiled content, such as:
+* Transpiled JavaScript, such as TypeScript or minified JavaScript.
+* Compiled CSS, such as SASS or SCSS.
+
+WebView2 doesn't load source maps that are referenced by content which was loaded by using the `WebResourceRequested` event.
+
+For example, suppose you load `main.js` in your `WebResourceRequested` event handler by setting the `Response` property of `CoreWebView2WebResourceRequestedEventArgs`.  If `main.js` references `main.js.map` as its source map:
```

Comment:
Win32 EventArgs has only a getter, not a setter.  Is this the wrong API, or is Win32 Ref docs missing ::set_Response?  .NET has getter/setter.

.NET:  get/set
https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2webresourcerequestedeventargs#properties

WinRT:  get/set
https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2webresourcerequestedeventargs#response


Win32:  get, no set
https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2webresourcerequestedeventargs#get_response

@dhveerap @mikosg